src/runtime_src/hip/core/error.h

```diff
@@ -7,42 +7,166 @@
 #include "core/common/api/bo.h"
 
 #include "hip/config.h"
+#include "hip/core/device.h"
 #include "hip/hip_runtime_api.h"
 
+#include "xrt/experimental/xrt_error.h"
+
 namespace xrt::core::hip
 {
+  /**
```

Comment:
Do not indent within a namespace.

```suggestion
namespace xrt::core::hip {
/**
```